### ISSUE TITLE
Is there any CuDNN bindings update plan?

### ISSUE THREAD

**Issue by hfxunlp** [soumith/cudnn.torch#378]
I love this library because it is quite fast and efficient. Nvidia is going to release cuda 9 and cudnn 7 which should be faster, so I hope this library can keep up-to-date with nvidia.

**Comment by borisfom** [comment]
@ngimel : should we submit a PR?

**Comment by ngimel** [comment]
Yes, it should go into new branch.

**Comment by borisfom** [comment]
Right. @soumith : can you please create a branch for CUDNN7 ?

**Comment by ngimel** [comment]
Done, I've created off R6.

**Comment by borisfom** [comment]
Submitted https://github.com/soumith/cudnn.torch/pull/379

**Comment by hfxunlp** [comment]
Thank you for your work @borisfom @ngimel

**Comment by ngimel** [comment]
To be honest, you should not expect big performance gains unless you have Volta. Group convolution s and batch norm will be slightly faster.

**Comment by hfxunlp** [comment]
Thank you for telling me that. My friend has installed tensorflow with cudnn 6 on my workstation; At least I do not have to keep cudnn 5 anymore with these new branches. Updating packages has become more convenient, because the execution of update-alternatives is unnecessary now. @ngimel
